gcc/jit/jit-playback.cc

```diff
@@ -652,9 +766,31 @@ global_new_decl (location *loc,
   if (loc)
     set_tree_location (inner, loc);
 
+  set_variable_attribute (attributes, inner);
+
   return inner;
 }
 
+void
+playback::
+set_variable_attribute (const std::vector<std::pair<gcc_jit_variable_attribute,
+						    std::string>> &attributes,
+			tree decl)
+{
+  for (auto attr: attributes)
+  {
+    gcc_jit_variable_attribute& name = std::get<0>(attr);
+    std::string& value = std::get<1>(attr);
+    tree attribute_value = build_tree_list (NULL_TREE,
+	::build_string (value.length () + 1, value.c_str ()));
+    tree ident = get_identifier (variable_attribute_to_string (name));
+
+    DECL_ATTRIBUTES (decl) = tree_cons (ident,
```

Comment:
Do we know if we should call `decl_attributes` here as well, like we did for functions?

Reply:
We apparently don't have tests for variable attributes. :3

Reply:
We do now!